adafruit_requests.py

```diff
@@ -429,8 +430,8 @@ def iter_content(self, chunk_size: int = 1, decode_unicode: bool = False) -> byt
         self.close()
 
 
-class Session:
-    """HTTP session that shares sockets and ssl context."""
+class SocketHandler:
```

Comment:
Took all the `socket` based methods from `Session` and moved them to `SocketHandler`